bucket/foobar2000.json

```diff
@@ -15,24 +18,41 @@
             "Foobar2000"
         ]
     ],
+    "persist": [
+        "configuration",
+        "index-data",
+        "library",
+        ["playlists-v1.4","playlists"],
```

Comment:
Why 2 items here?

And what about `profile`?

Also, move the persist object below uninstaller

Reply:
> Why 2 items here?
> `["playlists-v1.4","playlists"],`

Reply:
@AkariiinMKII you missed my question